src/main/java/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -298,7 +298,13 @@ public void refreshTitle() {
       }
       getSupportActionBar().setDisplayHomeAsUpEnabled(true);
     } else {
-      title.setText(DcHelper.getConnectivitySummary(this, R.string.app_name));
+      String titleText;
+      if (DcHelper.getAccounts(this).getAll().length > 1) {
+        titleText = DcHelper.getConnectivitySummary(this, DcHelper.getContext(this).getName());
+      } else {
+        titleText = DcHelper.getConnectivitySummary(this, R.string.app_name);
```

Comment:
```suggestion
        titleText = DcHelper.getConnectivitySummary(this, getString(R.string.app_name));
```

i think, it is easier to move `getString()` here than maintaining a second signature of `getConnectivitySummary()` - i needed a moment to understand what is going on otherwise :)

Reply:
ah, i see, that's a reason for two signatures then, even if minor :)

let's go with the PR as is then